english/challenge-marshmellow-to-rock-off/tenacious-d.md

```diff
@@ -0,0 +1,30 @@
+Realizing now that the marshmallows have become self-aware, much unlike SkyNet, 
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.